site/content/3.12/release-notes/version-3.12/api-changes-in-3-12.md

```diff
@@ -11,6 +11,22 @@ archetype: default
 
 ### Behavior changes
 
+#### VelocyStream protocol removed
+
+ArangoDB's own bi-directional asynchronous binary protocol VelocyStream is no
+longer supported.
+
+You can no longer specify any scheme starting with `vst` in the
+`--server.endpoint` startup option of client tools.
```

Comment:
In the latest devel-nightly image (2024-02-06T20:14:15), it seems to still be possible to connect using `vst://`?!
On the other hand, it seems like it wasn't ever possible to bind the server to e.g. `vst+tcp://localhost:8529`.

What does the server do if the VST protocol is requested?

Reply:
Connecting via VST protocol is definitely not supported in the latest version of `devel`.
The server will immediately close any incoming connections that try to upgrade to/use the VST protocol.